src/PasswordReset.js

```diff
@@ -36,7 +36,11 @@ class PasswordReset {
             idBaseUrl: identityUrl,
         });
         this.clientSecret = this.client.generateClientSecret();
-        this.identityServerDomain = identityUrl.split("://")[1];
+        this.identityServerDomain = identityUrl ? identityUrl.split("://")[1] : null;
```

Comment:
I feel like we should use real url parsing now...

Reply:
In principle I agree but it barely seems worth changing it for a deprecated thing.

Reply:
Good point. Rest in peace, old password reset.